calculator/parser_test.py

```diff
@@ -0,0 +1,26 @@
+from parser import rpn
+from lexer import lex
+
+
+def test_basic():
+    assert rpn(lex("1 1 +")) == [2]
+
+
+def test_multi_level():
+    assert rpn(lex("3 6 + 7 *")) == [63]
```

Comment:
Do me a favor, when doing these asserts, follow this convention:
```
assert rpn(lex("3 6 + 7 *")) == (3 + 6) * 7
```
This way I know precisely what to expect the parser to have parsed.